features/FEATURE_LWIP/lwip-interface/mbed_lib.json

```diff
@@ -16,7 +16,11 @@
         "addr-timeout": {
```

Comment:
Is this left out for backward compatibility ? Why and how ? Isn't the lwipopts.h also updated ?